src/rigid_pendulum_poc/utilities.cpp

```diff
@@ -62,38 +63,39 @@ HostView2D create_identity_matrix(size_t size) {
     return matrix;
 }
 
-HostView1D create_vector(const std::vector<double>& values) {
-    auto vector = HostView1D("vector", values.size());
-    auto entries = Kokkos::RangePolicy<Kokkos::DefaultHostExecutionSpace>(0, values.size());
-    auto fill_vector = [vector, values](int index) {
-        vector(index) = values[index];
-    };
+Kokkos::View<double*> create_vector(const std::vector<double>& values) {
+    auto vector = Kokkos::View<double*>("vector", values.size());
+    auto vector_host = Kokkos::create_mirror(vector);
 
-    Kokkos::parallel_for(entries, fill_vector);
+    for (unsigned index = 0; index < values.size(); ++index) {
```

Comment:
Should we consider using `size_t` as the type of `index` here (and other instances below)? Looks like it's more aligned with [idiomatic use of C++](https://stackoverflow.com/questions/131803/unsigned-int-vs-size-t).